<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Poppins;
background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QTableWidget {
	border: 0;
}

QPushButton {
	height: 40px;
	border-radius: 20px;
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
	font-weight: 500;
	font-size: 18px;
}</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
	font-size: 14px;
}

QHeaderView::section {
	font-size: 14px;
	font-weight: bold;
	border: 1px solid #d9d9d9;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <property name="columnCount">
     <number>13</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Etiqueta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Temperatura</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OD</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TDS</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>pH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Calidad</string>
     </property>
    </column>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string>Valledupar</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>31.92</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>14.84</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>457.30</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>9.65</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>914.6</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>Manaure</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>25.37</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>7.75</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>526.49</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>11.98</string>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string>1052.98</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>La Paz</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>28.80</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>21.96</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>706.50</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>7.89</string>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string>1413.00</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>195</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{ 
    background-color: rgb(234, 234, 234);
    height: 20px;
    border-radius: 10px;
}

QSlider::handle:horizontal { 
    background-color: #00007f;
    width: 60px;
    height: 20px;
    margin-left: 0px; 
    margin-right: 0px;
    border-radius: 10px;
}

QSlider{
    background-color: rgb(234, 234, 234);
    border-radius: 10px;
}
</string>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPageBtn">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prevPageBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dataCountLbl">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10pt;</string>
    </property>
    <property name="text">
     <string>1-5 de 11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graphBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e63946;
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
